### DISCUSSION TITLE
7/24 진척사항 관리

### DISCUSSION THREAD

**Discussion by myrhymetree** [goalapa/spring-cacamuca#130] [General]
회의 목적 : 배포 전까지 기능 완료 가능한지 점검하고, 맡은 기능 우선순위 정하기

회원(성준)
- 회원가입 완료
- 로그인 완료
- 회원정보변경 완료
- 회원탈퇴 완료
- 회원목록조회 완료
- 아이디찾기/비밀번호찾기 완료
테스트코드 작성중(24일~25일 사이에 완료)

요청(정인)
- 이미지 오류 제외하고 완성
- 통계
- 식품 별 통계 정보(리뷰 개수, 좋아요 개수, 평균 별점, 선호 나이대, 선호 성별) 완료
국가별 순위 보여주는 작업 하는중

리뷰(현석)
- 리뷰 작성 완료
- 리뷰 조회(나라+음식이름) 완료
- 좋아요, 신고 기능 완료
- 리뷰 검색 (논의 필요) // 메인페이지 검색바에서 식품 명 검색 시 해당 식품의 리뷰가 나오는것으로 논의완료
- 검색할때 검색 키워드와 관련된 음식이 나오는지? 리뷰가 나오는지?
검색 및 상세 조회 프론트 작업 중

카테고리(주원)
카테고리 수정(80% 진행)

식품(주원)
- 메인페이지 프론트(20% 진행)
~~검색(0% 진행)~~ 검색은 리뷰에서 처리하는것으로 논의완료
- 등록, 수정, 삭제 기능 구현 진행 중
식품 등록, 수정, 삭제기능을 먼저 진행하는것으로 논의 완료함, 메인페이지는 후순위

신고(수의)
- 신고 리스트 조회 완료
- 신고하기 완료
- 신고 리스트에서 삭제 완료
- 블랙리스트 조회
- 누적신고횟수에 따라 블랙리스트 생성
- 블랙리스트 해제

블랙리스트 진행 중, 신고에서 리뷰 번호 받아서 신고 진행시키는 것 진행 중

검색 기능만 변경사항 있고, 7/26일까지 나머지 기능 완료할 수 있는것으로 논의됨.

**Comment by myrhymetree** [comment]
@numerical43 @Dylan-SonJungin @bbbbooo @juunewon 오늘 회의 내용 정리했습니다.

**Reply by bbbbooo** [reply]
확인했습니다
